js/clinicalTimeline.js

```diff
@@ -156,7 +155,7 @@ var clinicalTimeline = (function(){
     d3.select(divId + " svg")
       .insert("rect", ".timeline-label")
       .attr("width", 130)
-      .attr("height", gBoundingBox.height - 15)
+      .attr("height", 245)
```

Comment:
The height of the timeline is dynamically determined (https://github.com/cBioPortal/clinical-timeline/blob/master/js/clinicalTimeline.js#L123) and thus we can't maintain a constant for the same. Hence I have used `svg.attr("width")`